src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultOfDivision = {};
+  const elementForDivision = sourceString.split(':').map(item => item.trim())
+    .join(': ').split(';').map(style => style.trim())
+    .filter(value => value.length > 1);
```

Comment:
```suggestion
  const elementForDivision = sourceString
    .split(':')
    .map(item => item.trim())
    .join(': ')
    .split(';')
    .map(style => style.trim())
    .filter(value => value.length > 1);
```
так немного полегче читается